test/common/wpt.js

```diff
@@ -411,7 +411,7 @@ class StatusLoader {
         const list = this.grep(filepath);
         result = result.concat(list);
       } else {
-        if (!(/\.\w+\.js$/.test(filepath))) {
+        if (!(/\.\w+\.js$/.test(filepath)) || filepath.endsWith('.helper.js')) {
```

Comment:
Shouldn't it be a dash rather than a dot?
```suggestion
        if (!(/\.\w+\.js$/.test(filepath)) || filepath.endsWith('-helper.js')) {
```

Reply:
No, it shouldn't.